draft-ietf-scitt-architecture.md

```diff
@@ -218,7 +218,9 @@ Equivocation:
 
 Feed:
 
-: see Subject
+: A collection of Signed Statements, observed by the Transparency Service, related to the `sub` field of the 'CWT_Claims' property of the protected header of Signed Statements.
+
+Verifiers use the Feed to ensure completeness and Non-equivocation in supply chain evidence by identifying all Transparent Statements linked to the Artifact they are evaluating.
```

Comment:
```suggestion
```

Reply:
A feed returns the collection of signed statements (or transparent statements). It returns what's on the append-only log. 
It simply returns what the transparency service persisted.

Reply:
See https://github.com/ietf-wg-scitt/draft-ietf-scitt-architecture/pull/176/commits/e674b8c1c9b19dbbbe8fc75dd4ac993670438e3d for clarification 

> Feed:
>
> : A collection of Receipts, as recorded by the Transparency Service, based on filtering of properties from the envelope including, but not limited to the `sub` field of the `CWT_Claims`.